packages/db/scripts/processing/add_metadata_to_lesson_summaries.ts

```diff
@@ -1,39 +1,42 @@
+import { aiLogger } from "@oakai/logger";
+
 import { prisma } from "../../index";
 
+const log = aiLogger("db");
+
+const processBatch = async (skip: number) => {
```

Comment:
Fixes a code complexity warning too

Reply:
Great! I always appreciate a change like this